.github/workflows/deployment.yml

```diff
@@ -0,0 +1,45 @@
+# This is a basic workflow to help you get started with Actions
+#python -m pip install --upgrade pip
+#pip install setuptools wheel twine
+
+
+name: deployment
+
+#Runs on every push (PR) to master
+on:
+  push:
+    branches:
+      - master
+      - deploytest
+
+jobs:
+  deployment:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+      - name: FTP Deploy
+        uses: SamKirkland/FTP-Deploy-Action@3.1.1
+        with:
+          # Deployment destination server & path. Formatted as protocol://domain.com:port/full/destination/path/
+          ftp-server: ${{ secrets.DEPLOYMENT_DESTINATION }}
+          ftp-username: ${{ secrets.SFTP_USERNAME }}
+          ftp-password: ${{ secrets.SFTP_PASSWORD }}
+          git-ftp-args: --insecure
+      - name: SSH Remote Commands
+        uses: appleboy/ssh-action@v0.1.3
+        with:
+          host: ${{ secrets.HOST }}
+          username: ${{ secrets.SFTP_USERNAME }}
+          password: ${{ secrets.SFTP_PASSWORD }}
+          script: |
+            ps aux | grep python | grep -v "grep python" | awk '{print $2}' | xargs kill -9
+            cd /srv/sam-bot
+            python3 -m pip install --upgrade pip
+            python3 -m venv env
```

Comment:
I would like to name the directory for the virtual environment `venv` instead of `env` if that's possible.